browser/app/js/browser/__tests__/ChangePassword.test.js

```diff
@@ -44,11 +44,11 @@ describe("ChangePasswordModal", () => {
   }
```

Comment:
Similar to `AboutModal`, we should change the name on the `describe` block of the test (line 37) from `ChangePasswordModal` to `ChangePassword`.